config.js

```diff
@@ -38,4 +38,13 @@ try {
 	//it is okay if no override file is provided
 }
 configInput.colors = getChalkColors(configInput, providedConfig);
+configInput._logLevels = {

```

Comment:
these levels should be reviewed, not sure if they actually make sense, I was just trying to get functionality down this morning.